datafusion/core/src/physical_plan/windows/bounded_window_agg_exec.rs

```diff
@@ -123,7 +122,7 @@ impl BoundedWindowAggExec {
         let mut result = vec![];
         // All window exprs have the same partition by, so we just use the first one:
         let partition_by = self.window_expr()[0].partition_by();
-        let sort_keys = self.sort_keys.as_deref().unwrap_or(&[]);
+        let sort_keys = self.input.output_ordering().unwrap_or(&[]);
```

Comment:
this makes sense to me -- specifically that `BoundedWindowExec` does not change the ordering of its input and thus there is no reason to store the sort key directly